dashboard/modules/job/tests/test_job.py

```diff
@@ -1,23 +1,165 @@
 import os
 import sys
+import copy
 import time
 import logging
 import requests
+import tempfile
+import zipfile
+import shutil
 import traceback
 
 import ray
 from ray._private.utils import hex_to_binary
 from ray.new_dashboard.tests.conftest import *  # noqa
+from ray.new_dashboard.modules.job import job_consts
 from ray.test_utils import (
     format_web_url,
     wait_until_server_available,
+    wait_for_condition,
 )
 import pytest
 
-os.environ["RAY_USE_NEW_DASHBOARD"] = "1"
-
 logger = logging.getLogger(__name__)
 
+JOB_ROOT_DIR = "/tmp/ray/job"
```

Comment:
use `tempfile.TemporaryDirectory` for windows compatibility.

Reply:
I updated the test case to get the job root directory from the session dir.